tests/drivers/at/main.c

```diff
@@ -370,6 +436,9 @@ int main(void)
 {
     puts("AT command test app");
 
+    initialized = false;
+    is_power_on = false;
```

Comment:
I am not sure about this PR

but for initialization you should just use the initializer instead of setting the value at the start of the program

l 36 and l 37 

false (0) should also be the default initializer, in c static values are initialized (unless you tell the compiler it should not) -> not writing the initializer might also be OK

Reply:
I move initialization form `main` function to the static variable initializer.

Reply:
Checking for initialization is fine, since using the `at_*` methods without proper initialization would be UB. I'm not sure about the power state tho. What if we want to test an error case, e.g. proper command timeout if the module is powered off?